### PR TITLE
Fix typo and link to your blog

### PR DESCRIPTION
**Pull request by kronik3r**
Hallo!
I appreciate your openness, I hope other companies will get inspired by it.
I was reading your document while making an application and noticed a typo and a missing link. 👍

Fix typo in hiring_process section
Add link in what_love_picnic section

### CODE REVIEW

File: src/pages/what_love_picnic.adoc [hunk 1]
```diff
@@ -37,7 +37,7 @@ belongs to you as much as to anyone else.
 
 Code reviews are at the core of our development, but it's not always
 fun. We want everything to be interesting so we've put code reviews on
-the blockchain. You can find out more in this article.
+the blockchain. You can find out more in https://blog.picnic.nl/crypto-incentives-for-code-reviews-71a0be53d130[this article].
```
Comment (Stephan202) [inline src/pages/what_love_picnic.adoc]: We have started to externalize our links. IIUC we can add this one to `links.adoc`. A link ID could be `:url-picnic-blog-incentives-for-code-reviews:`

**Review by Stephan202 — COMMENTED** [review]
Tnx!

File: src/pages/what_love_picnic.adoc (hunk 1 above)
Comment (kronik3r) [reply src/pages/what_love_picnic.adoc]: @Stephan202 resolved 👍

File: src/links.adoc [hunk 2]
```diff
@@ -1 +1,2 @@
 :url-picnic-hq-gmaps: https://www.google.com/maps/search/Picnic+HQ/@52.3314517,4.9166934,17z[Picnic HQ]
+:url-picnic-blog-incentives-for-code-reviews: https://blog.picnic.nl/crypto-incentives-for-code-reviews-71a0be53d130[this article]
```
Comment (Mordavolt) [inline src/links.adoc]: The current name is not really reusable, I suggest renaming it.
```suggestion
:url-picnic-blog-incentives-for-code-reviews: https://blog.picnic.nl/crypto-incentives-for-code-reviews-71a0be53d130[Crypto incentives for code reviews]
```

**Review by Mordavolt — COMMENTED** [review]
One last thing :)

**Review by Mordavolt — APPROVED** [review]
Awesome, thank you for your time and commitment :)

**Comment by kronik3r** [comment]
@Mordavolt Glad to be of help! :)

**Pull request merged**